app/views/pages/get_started.html.erb

```diff
@@ -0,0 +1,73 @@
+<% set_page_title("") %>
```

Comment:
Is this supposed to be blank?

Reply:
Woops, no, I forgot to go back and check what the page title is supposed to be. Thanks for catching that!

@hannahkc do you have a page title in mind for this page? I couldn't see one on the google doc.

Reply:
Same pattern as the other pages I reckon? So: Get started - GOV⁠.⁠UK Forms

That sound ok?

Reply:
Done!